pages/browse/events.vue

```diff
@@ -0,0 +1,29 @@
+<script lang="ts" setup>
+import type { IAllDocumentModels } from "@bcc-code/bmm-sdk-fetch";
+
+const { data: models, pending } = useBrowseEvents();
+
+toolbarTitleStore().setReactiveToolbarTitle(() => models.value?.title ?? " ");
```

Comment:
@SimonSimCity @sifferhans 
The title is translated on the server.
But that means I don't have a title until I get the response from the server. How can I set the toolbar title?

Reply:
moved it to DocumentList.vue